crypto/bio/bio_addr.c

```diff
@@ -238,7 +238,18 @@ static int addr_strings(const BIO_ADDR *ap, int numeric,
             } else
 # endif
             {
+#if !defined(_WIN32_WCE)
                 ERR_raise_data(ERR_LIB_BIO, ERR_R_SYS_LIB, gai_strerror(ret));
+#else
+                wchar_t *wstr_error = gai_strerrorW(ret);
+                size_t length = wcslen(wstr_error) + 1;
+                char *str_error = OPENSSL_malloc(length);
+                if (str_error != NULL) {
```

Comment:
Line break between the declarations and this line please.

Reply:
Modified the code as you have suggested. Please check.